docs/guides/mobile-plugin-ima-ios.md

```diff
@@ -29,25 +29,23 @@ This plugin is in the following open-source GitHub repository:
 
 To run this plugin, install the following:
 
-1. [Google IMA SDK v3.18.5](https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side/history)
+1. [Google IMA SDK v3.19.1](https://developers.google.com/interactive-media-ads/docs/sdks/ios/client-side/history)
 1. [UID2 SDK for iOS v0.2.0](../sdks/uid2-sdk-ref-ios.md)
-1. UID2 IMA Plugin for iOS v0.1.0
+1. UID2 IMA Plugin for iOS v0.2.0
 
 ## Installation
 
 Prerequisite: Install the Google Interactive Media Ads SDK and the UID2 iOS SDK.
 
-Install the UID2 iOS IMA Plugin v0.1.0 via Swift Package Manager to an existing app with the UID2 iOS SDK and the Google Interactive Media Ads SDK installed.
+Install the UID2 iOS IMA Plugin v0.2.0 via Swift Package Manager to an existing app with the UID2 iOS SDK and the Google Interactive Media Ads SDK installed.
```

Comment:
@bleege could you confirm if IMA plugin can be installed via Swift Package Manager?

Reply:
No only can the IMA Plugin be installed via Swift Package Manager, it's the recommended way.

Reply:
No edit here, I believe. LMK if there is. Thx.